pykdl_ros/test/test_import.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+
+class TestImport(unittest.TestCase):
```

Comment:
This seems like a bit of a brief test (although there's hardly any functionality)

Reply:
Just a simple test to check that importing works, so all deps are installed and no circular imports etc.